flair/embeddings/transformer.py

```diff
@@ -1353,6 +1353,11 @@ def from_params(cls, params):
 
     def to_params(self):
         config_dict = self.model.config.to_dict()
+
+        # do not switch the attention implementation upon reload.
+        config_dict["attn_implementation"] = self.model.config._attn_implementation
+        config_dict.pop("_attn_implementation_autoset", None)
```

Comment:
This is unrelated to this PR but necessary to get CI to pass. It's based on helpmefindaname's [commit](https://github.com/flairNLP/flair/pull/3560/commits/783ebc7c5db43f2d0e3ca0d6473556da34a4e70e).